fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckAction.kt

```diff
@@ -87,5 +92,55 @@ sealed interface ReviewQualityCheckAction : Action {
     /**
      * Triggered when opening a link from the review quality check feature.
      */
-    data class OpenLink(val link: ReviewQualityCheckState.LinkType) : NavigationMiddlewareAction
+    data class OpenLink(val link: ReviewQualityCheckState.LinkType) : NavigationMiddlewareAction, TelemetryAction
+
+    /**
+     * Triggered when the bottom sheet onboarding card is displayed.
+     */
+    object OnboardingDisplayed : TelemetryAction
+
+    /**
+     * Triggered when the bottom sheet is closed.
+     */
+    object BottomSheetClosed : TelemetryAction
+
+    /**
+     * Triggered when the bottom sheet is opened.
+     */
+    object BottomSheetDisplayed : TelemetryAction
+
+    /**
+     * Triggered when the user clicks on the "Not now" button from the contextual onboarding card.
+     */
+    object NotNowClicked : TelemetryAction
+
+    /**
+     * Triggered when the user expands the recent reviews card.
+     */
+    object ShowMoreRecentReviewsClicked : TelemetryAction
+
+    /**
+     * Triggered when the user expands the settings card.
+     */
+    object ExpandSettingsClicked : TelemetryAction
+
+    /**
+     * Triggered when the No analysis card is displayed to the user.
+     */
+    object NoAnalysisDisplayed : TelemetryAction
+
+    /**
+     * Triggered when the user clicks on the analyze button
+     */
+    object AnalyzeClicked : TelemetryAction
+
+    /**
+     * Triggered when the user clicks on the reanalyze button
+     */
+    object ReanalyzeClicked : TelemetryAction
```

Comment:
Can used `ReanalyzeProduct` action here. Similarly, let's rename the one about to `AnalyzeProduct`.